README.md

```diff
@@ -43,12 +48,94 @@ In your terminal, navigate to `<your-project>/wp-content/themes/flynt-starter-th
 
 Changes to files made in `Components` and `Features` will now be watched for changes and compiled to the `dist` folder.
 
-**The full documentation is coming soon.**
+### Configuring Page Templates
+All template files can be found under the theme root, in the `templates` directory.
+
+The structure of each page within the theme is created using a nested JSON object. Each PHP template within the `templates` directory takes a simple JSON configuration file, and using the Flynt Core plugin, parses and renders this into HTML.
+
+For example, take `templates/page.php`:
+
+```php
+<?php
+
+Flynt\echoHtmlFromConfigFile('default.json');
+```
+
+The JSON template configuration files are found in `config/templates`. These configuration files define the [areas](#areas) and [components](#components) which are loaded into the template.
+
+Take `config/templates/default.json` as an example. This template calls the `DocumentDefault` component and registers one area: `layout`. The `layout` area calls the `LayoutSinglePost` component, which in turn contains three nested [areas](#areas): `mainHeader`, `pageComponents`, and `mainFooter`. In addition, the `pageComponents` area calls the `ComponentLoaderFlexible` component.
```

Comment:
i am not really happy with `calls` and `registers`. components are registered somewhere else. and not called. What about contains, consists of, is made up of / is composed with?. I am sure you have some ideas ;)